doc/1/controllers/collection/truncate/index.md

```diff
@@ -0,0 +1,29 @@
+---
+code: true
+type: page
+title: TruncateAsync
+description: Remove all documents from collection.
+---
+
+# TruncateAsync
+
+Remove all documents from a collection while keeping the associated mapping.  
+It is faster than deleting all documents from a collection.
```

Comment:
Actually, it's not. It's exactly as fast as using `deleteByQuery` with an empty filters object, since this is what's done by our current code.

We removed that line from our API documentation, and I think we should do the same here:

```suggestion
```